config/environments/production.rb

```diff
@@ -34,37 +34,31 @@
                          [:memory_store, { size: 64.megabytes }]
                        end
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV['RAILS_MASTER_KEY']` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
We need to understand how Rails 6 / Webpack(er) / Heroku works for our JS. We have very little, but it needs to still work.

Reply:
Is sprocket completely remove from rails 6, or we can continue to use it since the project didn't start with webpacker ?

Reply:
I'm not sure.

Reply:
it looks like this is the relevant PR to read through `https://github.com/rails/rails/pull/33079`